ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Yeah, I didn't think about the existing transfer/common. I'll look for a solution